tools/docker/Dockerfile.integ-test

```diff
@@ -30,18 +30,30 @@ RUN mkdir -p \
   ${FICD_LOG_DIR} \
   /etc/cni/net.d
 
-# Install containerd to have runc shim.
-ENV CTRD_VERSION="1.6.4"
+# Install containerd to have runc shim for amd.
+ENV CTRD_VERSION="1.6.8"
 RUN wget --quiet -O- https://github.com/containerd/containerd/releases/download/v$CTRD_VERSION/containerd-$CTRD_VERSION-linux-amd64.tar.gz | tar zxf - -C /tmp/ && \
   install -D -o root -g root -m755 -t /usr/local/bin /tmp/bin/containerd-shim-runc-v2 && \
   rm -rf /tmp/bin
 
-# Install critest.
+# Install containerd to have runc shim for arm.
+ENV CTRD_VERSION="1.6.8"
+RUN wget --quiet -O- https://github.com/containerd/containerd/releases/download/v$CTRD_VERSION/containerd-$CTRD_VERSION-linux-arm64.tar.gz | tar zxf - -C /tmp/ && \
+  install -D -o root -g root -m755 -t /usr/local/bin /tmp/bin/containerd-shim-runc-v2 && \
+  rm -rf /tmp/bin
+
+# Install critest for AMD.
 ENV CRITEST_VERSION="1.23.0"
 RUN wget --quiet -O- https://github.com/kubernetes-sigs/cri-tools/releases/download/v$CRITEST_VERSION/critest-v$CRITEST_VERSION-linux-amd64.tar.gz | tar zxf - -C /tmp/ && \
   install -D -o root -g root -m755 -t /usr/local/bin /tmp/critest && \
   rm -f /tmp/critest
 
+# Install critest for ARM.
+ENV CRITEST_VERSION="1.23.0"
+RUN wget --quiet -O- https://github.com/kubernetes-sigs/cri-tools/releases/download/v$CRITEST_VERSION/critest-v$CRITEST_VERSION-linux-arm64.tar.gz | tar zxf - -C /tmp/ && \
+  install -D -o root -g root -m755 -t /usr/local/bin /tmp/critest && \
+  rm -f /tmp/critest
+
```

Comment:
same suggestion as above - this will overwrite the critest binary installed for x86.